### ISSUE TITLE
Cannot set async to false

### ISSUE THREAD

**Issue by rossmd** [Topener/XHR#13]
When trying to set async to false, the code will always set it to true.  Look at this line:

```
line 31: extraParams.async = extraParams.async || true;
```

If I pass in a value of false for async, that line is evaluated as:

```
extraParams.async = false || true;
```

which will always be true.  Perhaps you need something like:

```
extraParams.async = (extraParams.hasOwnProperty('async')) ? extraParams.async : true
```

**Comment by raulriera** [comment]
you are right, if you have a pull request please send it this way :) otherwise I will upload a fix after some testing

**Comment by rossmd** [comment]
I'm very new to git, so I attempted to send a pull request.  If I screwed something up, please let me know.

**Comment by raulriera** [comment]
Fix in commit 413201a
